server/jobHandler/src/isJobComplete.ts

```diff
@@ -39,7 +39,7 @@ export const isJobComplete = async (key: string): Promise<any> => {
 
 	//Iterate though every file and check the value of "finished"
 	for (const element of res.files) {
-		ready &&= (element.status == FileStatus.DONE);
+		ready && (element.status == FileStatus.DONE);
```

Comment:
the &&= is in the main branch so idk if its correct or it should be == ; lmk if I need to change it

Reply:
&&= is in the same family of operators as +=, *=, etc..
It's equivalent to `ready = ready && (element.status == FileStatus.DONE);`, and was used here intentionally.

Reply:
Okay no stress I only asked because I thought you (Peter) said to remove when we were looking at my code last and it was coming up with an error, but I'll fix it